splunkrepl.js

```diff
@@ -243,96 +257,163 @@ function doQuery(query, callback) {
     var search = 'search ' + query;
     self.lastSearch = query;
     Async.chain([
-        function(done) {
-            self.service.login(done);
-        },
-        function(success, done) {
-            self.service.search(search, {}, done);
-        },
-        function(job, done) {
-            job.track({}, function(job) {
-                job.results({}, done);
-            });
-        },
-        function(results, job, done) {
-            if (results.rows.length == 0) {
-                if (useJson) {
-                    console.log("[]");
-                    return done();
+            function (done) {
+                self.service.login(done);
+            },
+            function (success, done) {
+                searchParams = realtime ? {
+                    earliest_time: 'rt',
```

Comment:
What if someone want to specify the earliest, latest? You can do this within the query via SPL. I am guessing this will override this and force it always to rt.